plugin.go

```diff
@@ -44,6 +44,9 @@ func (p Plugin) Exec() error {
 
 	var commands []*exec.Cmd
 	remote := p.Config.Remote
+	if len(p.Config.Secrets) != 0 {
+		exportSecrets(p.Config.Secrets)
```

Comment:
Just want to link this to https://github.com/drone/drone/issues/1888 which proposes the plugin attribute idea